packages/client/test/integration/waitForStorage.test.ts

```diff
@@ -69,7 +69,7 @@ describe('waitForStorage', () => {
         await stream.addToStorageNode(storageNode.getAddress())
         const msg = convertStreamMessageToMessage(await createSignedMessage({
             messageId: new MessageID(stream.id, 0, Date.now(), 0, PUBLISHER_ID, 'msgChainId'),
-            serializedContent: JSON.stringify({}),
+            serializedContent: utf8ToBinary(JSON.stringify({})),
```

Comment:
```suggestion
            serializedContent: new Uint8Array()
```

maybe this change can be done in all places where we use `content/serializedContent: utf8ToBinary(JSON.stringify({}))` or `content/serializedContent: utf8ToBinary('{}')`